src/xrScriptEngine/script_engine.cpp

```diff
@@ -524,7 +524,7 @@ struct raii_guard : private Noncopyable
 #ifdef DEBUG
             static const bool break_on_assert = !!strstr(Core.Params, "-break_on_assert");
 #else
-            static const bool break_on_assert = true;
+            static const bool break_on_assert = false;	// xxx: true will hide real lua error, LuaJIT problem?
```

Comment:
Cuz with TRUE you always get the same log on every lua-crash:
```
FATAL ERROR
 
[error] Expression    : !m_error_code
[error] Function      : raii_guard::~raii_guard
[error] File          : script_engine.cpp
[error] Line          : 531
[error] Description   : see call_stack for details!
```
Game will crash with log anyway, regardless of break_on_assert being true or false. This code related only to Lua anyway.